lib/purge.js

```diff
@@ -0,0 +1,97 @@
+const axios = require('axios')
+const chalk = require('chalk')
+const creds = require('./config')
+const fs = require('fs')
+const prompt = require('prompt')
+const urljoin = require('url-join')
+
+prompt.message = ''
+
+module.exports.purge = async(force) => {
+  const config = creds.readConfig()
+  if (!checkDpIsThere()) {
+    console.error('Aborting: datapckage.json not found!')
+    process.exit(1)
+  }
+  const dpjson = JSON.parse(fs.readFileSync('datapackage.json').toString())
+  let schema = {
+    properties: {
+      'Package Name': {
+        pattern: `${dpjson.name}`,
+        message: `Did not match with package name. Type ${chalk.green(dpjson.name)} or re-run this command with --force`,
+        required: true
+      }
+    }
+  }
+  let warning = `${chalk.yellow('Warning')}: This action CANNOT be undone! This will permanently delete ${chalk.green(dpjson.name)} package from DataHub
+
+Please type in the name of the package to confirm`
+
+  if (!force) {
+    console.log(warning+'\n')
+    prompt.start()
+    prompt.get(schema, async (err, result) => {
+      if (err) {
+        return console.error(err.message+'\n');
+      }
+      const token = await getToken(config)
+      await requestPurge(config, dpjson.name, token)
+      console.log(`Successfully deleted ${dpjson.name}`)
+    })
+  } else {
+    const token = await getToken(config)
```

Comment:
I repeat myself here and doing same exact thing twice but could not find any better way to await prompt. Feel free to add commit fixing this